dropwizard-metrics/4.0/README.md

```diff
@@ -14,7 +14,19 @@ It is designed to be used with [the 4.0.0-SNAPSHOT of Dropwizard Metrics](https:
 
 ### Setting up Maven
 
-You will need both the DropWizard `metrics-core` and the Wavefront `metrics-wavefront` libraries as dependencies. Logging depends on `org.slf4j`:
+DropWizard has not yet released a 4.0.0 version meaning you can't yet use this directly from Maven Central. Instead you must first build the DropWizard package locally. This is pretty straightforward:
```

Comment:
It might be more futureproof to say "Until DropWizard releases 4.0.0...

That way, if someone is reading our docs in the future at a time when dropwizard has made the release, they will be less confused